Mathlib/SetTheory/Cardinal/Cofinality.lean

```diff
@@ -219,7 +219,7 @@ theorem cof_eq_sInf_lsub (o : Ordinal.{u}) : cof o =
     have := typein_lt_self a
     simp_rw [← hf, lt_lsub_iff] at this
     cases' this with i hi
-    refine ⟨enum (· < ·) (f i) ?_, ?_, ?_⟩
+    refine ⟨@enum _ (· < ·) o.out.wo ⟨f i, ?_⟩, ?_, ?_⟩
```

Comment:
Can you make it so that you don't need `@`? eg by using a keyed argument `(_ := _)`?

Reply:
I should mention that something I want to do in the near future is make the type argument of `typein` and `enum` explicit. That should make them much nicer to use in general.

Reply:
Okay, I won't bother you more with those `@` then